src/cdk-experimental/testing/test-app/protractor.e2e.spec.ts

```diff
@@ -0,0 +1,183 @@
+import {browser, by, element} from 'protractor';
+
+import {getElementFinder, load} from '../protractor';
+import {MainComponentHarness} from './harnesses/main-component-harness';
+
+describe('Protractor Helper Test:', () => {
+  let harness: MainComponentHarness;
+
+  beforeEach(async () => {
+    await browser.get('/');
+    harness = await load(MainComponentHarness, 'main');
+  });
+
+  describe('Locator ', () => {
+    it('should be able to locate a element based on css selector', async () => {
+      const title = await harness.title();
+      expect(await title.text()).toEqual('Main Component');
+    });
+
+    it('should be able to locate all elements based on css selector',
+      async () => {
+        const labels = await harness.allLabels();
+        expect(labels.length).toEqual(2);
+        expect(await labels[0].text()).toEqual('Count:');
```

Comment:
I think that most of these assertions can be switched to use `toBe` instead of `toEqual`.